packages/common/src/config/env.ts

```diff
@@ -21,5 +21,8 @@ export default Object.keys(process.env)
       'process.env.LOCAL_SERVER': Boolean(LOCAL_SERVER),
       'process.env.STAGING': 'STAGING_BRANCH' in process.env,
       'process.env.VSCODE': Boolean(JSON.stringify(process.env.VSCODE)),
+      'process.env.SANDPACK': Boolean(
+        JSON.parse(process.env.SANDPACK || 'false')
```

Comment:
**Why do `Boolean(JSON.parse(...))` instead of `Boolean(JSON.stringify(...))` like above?**

`Boolean(JSON.stringify(...))` will always be truthy even if `process.env.VSCODE = "false"`. `JSON.parse(val || 'false')` will help support a number of valid environment variable values like:

```
SANDPACK=
SANDPACK=false
SANDPACK=0
SANDPACK=1
SANDPACK=true
```

Reply:
Very nice!